rainfall_project.py

```diff
@@ -1 +1,17 @@
-#filler line
+total_rainfall = 0
+
+list_of_months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Set", "Oct", "Nov", "Dec"]
+
+year = int(input("Enter the number of years: "))
+
+for years in range(year):
+    print(f"Year {years + 1}")
+    for month in range(12):
+        month_data = (float(input(f"Enter the rainfall amount for {list_of_months[month]}: ")))
+        total_rainfall += month_data
+
+average_rainfall = total_rainfall / (int(year)*12)

```

Comment:
So you don't need to cast `year` as an int because you already did that in line 5.

IN addition, if year's value had been manually set (For example if line 5 said: `year = 3`), once again you wouldn't need to cast it to int because it already is by default,